validator/rpc/wallet_test.go

```diff
@@ -311,30 +311,7 @@ func TestServer_WalletConfig(t *testing.T) {
 	})
 }
 
-func TestServer_ImportKeystores_FailedPreconditions_WrongKeymanagerKind(t *testing.T) {
```

Comment:
Derived wallets should be able to import keystores as they are merely wrappers over imported wallets